veggieseasons/lib/screens/details.dart

```diff
@@ -245,7 +245,7 @@ class _DetailsScreenState extends State<DetailsScreen> {
     final veggie = model.getVeggie(widget.id);
 
     return SizedBox(
-      height: 150,
+      height: 400,
```

Comment:
I'm not sure why this change was made. In the original design, just a portion of the image was shown, since it's enough to give some visual appeal to the popup but doesn't push the detailed information for the veggie too far down the page. With this change, the image can take up > 1/2 the screen, pushing the details quite far down.

Unless you have a strong reason for keeping this change, I'd recommend reverting it.

Reply:
<img width="666" alt="Screenshot 2020-03-03 at 12 40 28 AM" src="https://user-images.githubusercontent.com/29337604/75708957-cd501600-5ce7-11ea-9196-c8c1119cda17.png">

thank you @RedBrogdon  for review my PR. I changed the size of image because it is asked in issue.
If your are recommending then i revert changes

Reply:
Ah, got it. I think we have a miscommunication. 😄 

I wrote that note after talking with the colleague who wrote the original bug report. What he meant was that, ideally, we'd implement an animated header that grew when the user pulled the screen down, similar to this one:

https://stackoverflow.com/questions/54580218/stretchy-header-effect

For right now, I recommend just taking the image size change out, and we can merge the rest of the changes you've included. If you have time later to work on another change (and you're interested in learning how Flutter's slivers work), you could try to implement the stretchy header in another PR.

Reply:
reverted image size @RedBrogdon  thank you for your feedback . I will try to implement the stretchy header.

Reply:
It's a difficult bit of UI to get right, so it may take some time. 😄